src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -352,6 +354,14 @@ internal static LoggerFilterOptions CreateFilterOptions(LoggerFilterOptions regi
                 configuration.TelemetryProcessorChainBuilder.UseSnapshotCollector(options.SnapshotConfiguration);
             }
 
+            if (telemetryProcessorFactories.Any())
+            {
+                foreach (ITelemetryProcessorFactory telemetryProcessorFactory in telemetryProcessorFactories)
+                {
+                    configuration.TelemetryProcessorChainBuilder.Use(telemetryProcessorFactory.Create);
```

Comment:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/NETCORE/src/Shared/Implementation/TelemetryConfigurationOptionsSetup.cs#L104

ApplicationInsights adds processors to default sink. Not sure if its applicable here.